src/components/breadcrumbs/BreadcrumbsNav.tsx

```diff
@@ -0,0 +1,60 @@
+import { trpc } from "@/client/lib/trpc";
+import Breadcrumbs from "@mui/material/Breadcrumbs";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { SelectableForTable } from "zapatos/schema";
+import $breadcrumbs from "./Breadcrumbs.module.css";
+
+const BreadcrumbsNav = () => {
+  const router = useRouter();
+  const paths = router.asPath.split("/");
+
+  let personData:
+    | SelectableForTable<"student">
+    | SelectableForTable<"user">
+    | undefined;
+
+  if (paths[1] === "students" && paths[2]) {
+    const { data } = trpc.student.getStudentById.useQuery(
+      { student_id: paths[2] },
+      { enabled: Boolean(paths[2]), retry: false }
+    );
+    personData = data;
+  } else if (paths[1] === "staff" && paths[2]) {
+    const { data } = trpc.para.getParaById.useQuery(
+      { user_id: paths[2] },
+      { enabled: Boolean(paths[2]), retry: false }
+    );
```

Comment:
this will probably break at some point, one of React's rules is that you can never change the hooks you call after the first render

so instead, you probably want to remove the `if` condition and put the same condition in the `enabled` option of the queries

Reply:
Thanks for pointing it out @codetheweb! I fixed the logic with https://github.com/sfbrigade/compass/pull/268/commits/c779816aa46880c7a850b169b6d2315fa3535433 and https://github.com/sfbrigade/compass/pull/268/commits/30d07c426351639700082ddcfaaba1ce2a61b3b1. Please Review